mne/io/brainvision/brainvision.py

```diff
@@ -321,27 +323,32 @@ def _read_vmrk_events(fname, event_id=None, trig_shift_by_type=None):
             trigger = event_id[mdesc]
         else:
             try:
-                trigger = int(re.findall(r'[A-Za-z]*\s*?(\d+)', mdesc)[0])
+                trigger = int(re.findall(r'S([\s\d]{2}\d{1})', mdesc)[0])
             except IndexError:
                 trigger = None
-            if mtype.lower() in trig_shift_by_type:
-                cur_shift = trig_shift_by_type[mtype.lower()]
-                if cur_shift is not None:
-                    trigger += cur_shift
-                else:
-                    trigger = None
+        if mtype.lower() in trig_shift_by_type:
+            cur_shift = trig_shift_by_type[mtype.lower()]
+            if cur_shift is not None:
+                trigger += cur_shift
+            else:
+                # The trigger has been deliberately shifted to None. Do not
+                # add this to "dropped" so we do not warn about something
+                # that was done deliberately. Just continue with next item.
+                trigger = None
+                continue
         # FIXME: ideally, we would not use the middle column of the events
         # array to store the duration. A better solution would be using
         # annotations.
```

Comment:
this comment is still highly relevant...

next PR on this will not change the way we do things?

Reply:
The FIXME comment seems unrelated to the current PR. A fix for that would not change the way we read events.

The comment about "we do not warn if an event is deliberately dropped" describes my change (inserting the `continue`), which I think is a good idea.